charts/collabora-online/0.0.1/questions.yaml

```diff
@@ -0,0 +1,332 @@
+        #   schema:
+        #     type: string
+        #     default: ""    
+        # - variable: server_name
+        #   label: "Server Name (Basic setup probably won't need this)"
+        #   description: "When this environment variable is set (is not “”), then its value will be used as server name in /etc/loolwsd/loolwsd.xml. Without this, CODE is not delivering a correct host for the websocket connection in case of a proxy in front of it."    
+        #   schema:
+        #     type: string
+        #     default: ""
+        # - variable: DONT_GEN_SSL_CERT
+        #   label: "Don't generate SSL cert"
+        #   description: "When this environment variable is set (is not “”), then startup script will not generate a new SSL certificate signed by a dummy CA. It is useful, if you want to use your own SSL certificate for some reason."   
+        #   schema:
+        #     type: string
+        #     default: ""
+        
+  # Also adding storage makes install fail. -------------------
+
+  # # Storage
+  # # Configure app volumes
+  # - variable: appVolumeMounts
+  #   label: "app storage"
+  #   group: "Storage"    
+  #   schema:
+  #     type: dict
+  #     attrs:
+  #       # Config ------------------------
+  #       - variable: config
+  #         label: "Config Volume"
+  #         schema:
+  #           type: dict
+  #           $ref:
+  #             - "normalize/ixVolume"
+  #           attrs:
+  #             - variable: enabled
+  #               label: "Enabled"
+  #               schema:
+  #                 type: boolean
+  #                 default: true
+  #                 required: true
+  #                 hidden: true
+  #                 editable: false
+  #             - variable: emptyDir
+  #               label: "emptyDir"
+  #               schema:
+  #                 type: boolean
+  #                 default: false
+  #                 hidden: true
+  #                 editable: false
+  #             - variable: datasetName
+  #               label: "Name"
+  #               schema:
+  #                 type: string
+  #                 default: "config"
+  #                 required: true
+  #                 editable: false
+  #             - variable: mountPath
+  #               label: "Mount path inside the container"
+  #               schema:
+  #                 type: path
+  #                 required: true
+  #                 default: "/etc/loolwsd"
+  #                 editable: false
+  #             - variable: hostPathEnabled
+  #               label: "Config - Enable Host Path"
+  #               schema:
+  #                 type: boolean
+  #                 default: false
+  #                 show_subquestions_if: true
+  #                 subquestions:
+  #                   - variable: hostPath
+  #                     label: "Config Host Path"
+  #                     schema:
+  #                       type: hostpath
+  #                       required: true
```

Comment:
Possibly your container does not use s6_overlay and can't ready the mounted root volume.
This requires either a init container (hack) or a fix by IX systems to ix_volumes. I've reported the underlaying issue causing this to IX yesterday, expect some feedback on it next week and a fix before 21.04 ALPHA.

commenting it out isn't the way to go.

```suggestion
  # Storage
  # Configure app volumes
  - variable: appVolumeMounts
    label: "app storage"
    group: "Storage"    
    schema:
      type: dict
      attrs:
        # Config ------------------------
        - variable: config
          label: "Config Volume"
          schema:
            type: dict
            $ref:
              - "normalize/ixVolume"
            attrs:
              - variable: enabled
                label: "Enabled"
                schema:
                  type: boolean
                  default: true
                  required: true
                  hidden: true
                  editable: false
              - variable: emptyDir
                label: "emptyDir"
                schema:
                  type: boolean
                  default: false
                  hidden: true
                  editable: false
              - variable: datasetName
                label: "Name"
                schema:
                  type: string
                  default: "config"
                  required: true
                  editable: false
              - variable: mountPath
                label: "Mount path inside the container"
                schema:
                  type: path
                  required: true
                  default: "/etc/loolwsd"
                  editable: false
              - variable: hostPathEnabled
                label: "Config - Enable Host Path"
                schema:
                  type: boolean
                  default: false
                  show_subquestions_if: true
                  subquestions:
                    - variable: hostPath
                      label: "Config Host Path"
                      schema:
                        type: hostpath
                        required: true

```